understanding/22/consistent-help.html

```diff
@@ -62,18 +62,28 @@ <h2>Intent of Consistent Help</h2>
 
          <p>If the help item is visually in a different location, but in the same serial order, that is not helpful from a user's point of view, but it would not fail this criterion.</p>
 
-         <p>The location in a smaller viewport may be different than in a larger viewport, but it is best if the mechanism or link is consistent across a set of web pages. A consistent location, both visually and programmatically, is the most usable.</p>
-
          <p>When having problems completing a task on a Web site (or part of a Web site, what we call a <a>set of Web pages</a>), people with some types of disabilities may not be able to work through the issue without further help.  Issues could include difficulty:
          completing a form, or finding a document or page which provides information required to complete a task.</p>
 
          <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
 
-         <p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. PDFs and other static documents are not considered part of the "set of web pages" from which they are downloaded.</p>
+         <section id="limitations">
+            <h3>Limitations and Exceptions</h3>
+
+            <p>It is not the intent of this Success Criterion to require authors to provide help / access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. PDFs and other static documents are not considered part of the "set of web pages" from which they are downloaded.</p>
```

Comment:
light suggestion while this is being updated, but the lack of this instance of 'set of webpages' being linked, and the use of '/' instead of 'or' always struck me as odd.  but, no need to merge this if people disagree with my nits.

```suggestion
            <p>It is not the intent of this Success Criterion to require authors to provide help or access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in a consistent location. It does not require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. PDFs and other static documents are not considered part of the "<a>set of web pages</a>" from which they are downloaded.</p>
```